module/module.go

```diff
@@ -772,10 +735,35 @@ func (m *Module) AddStream(ctx context.Context, req *streampb.AddStreamRequest)
 	m.logger.CInfow(ctx, "AddStream calling AddTrack", "name", name, "tlsRTP", tlsRTP)
 	sender, err := m.pc.AddTrack(tlsRTP)
 	if err != nil {
-		return nil, errors.Wrap(err, "error adding track")
+		err = errors.Wrap(err, "error adding track")
+		if unsubErr := vcss.Unsubscribe(ctx, sub.ID); unsubErr != nil {
+			return nil, multierr.Combine(err, unsubErr)
+		}
+		return nil, err
```

Comment:
If `AddTrack` fails, we need to call `Unsubscribe` as otherwise the subscription will be orphaned.